docs/pages/database-access/reference/cli.mdx

```diff
@@ -295,8 +295,10 @@ $ tsh db login --db-user=postgres --db-name=postgres example
 
 | Flag | Description |
 | - | - |
-| `--db-user` | Optionally, set default database account name to connect as. |
-| `--db-name` | Optionally, set default database name to connect to. |
+| `--db-user` | The database user to log in as. |
+| `--db-name` | The database name to log in to. |
```

Comment:
Nit: is it "log in to", or "log into"?

Reply:
login to
log into
log in to

I googled it when I made the changes too lol. Proper form is "log in to"